backend/index.js

```diff
@@ -30,7 +31,7 @@ io.on('connection', (socket) => {
 
 // Start the server on our predetermined port number.
 server.listen(port, () => {
-    console.log('Listening on *:' + port);
+    console.log(`Listening on *:${port}`);
```

Comment:
I left the consoles to pass the test as we're still in the developing process and its being used for debugging purposes. I can change the config to throw warnings instead of errors so that we dont forget deleting them in the future